gammapy/estimators/energydependence.py

```diff
@@ -0,0 +1,262 @@
+import numpy as np
+from astropy.table import Table
+from gammapy.datasets import Datasets
+from gammapy.modeling import Fit
+from gammapy.modeling.models import FoVBackgroundModel
+from gammapy.modeling.selection import TestStatisticNested
+from gammapy.stats.utils import ts_to_sigma
+from .core import Estimator
+
+
+def weighted_chi2_parameter(results_edep, parameter="sigma"):
+    """Calculate the weighted chi2 value for the parameter of interest
+
+    Parameters
+    ----------
+    result_edep : dict
```

Comment:
The idea was to return a dict, similar to how it is done in the `~gammapy.modeling.select_nested_models`.
Open to other ideas, because this was a long debated topic. What are the most important values and how to show them!